app/utils/press-release-snapshot.js

```diff
@@ -53,6 +56,17 @@ export default class PressReleaseSnapshot {
     this.pressRelease.governmentFields = this.governmentFields;
     this.pressRelease.themes = this.themes;
     this.pressRelease.sources = this.sources;
+
+    //destroys and record which is added but not saved
+    if (this.pressRelease.attachments.length) {
```

Comment:
`this.pressRelease.attachments` best awaiten (en assignen aan een variable) voor je daarmee werkt. Op dit moment zal die promise onmiddellijk resolven, maar door de await toe te voegen maken we de code meer defensief mocht dat in de toekomst niet meer het geval zijn.

Reply:
Deze if-check is overbodig. `.forEach()` (of `.map()`) werken ook op een lege array